lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosAlgorithm.cxx

```diff
@@ -388,4 +407,3 @@ void FunctionalChaosAlgorithm::load(Advocate & adv)
 
 
 END_NAMESPACE_OPENTURNS
-
```

Comment:
you should save `weights`

Reply:
so no need to save it for `LeastSquaresExpansion`

Reply:
Fixed